.github/workflows/build-wheel-macos-arm64.yaml

```diff
@@ -213,30 +241,26 @@ jobs:
       id: cache-mhlo-build
       uses: actions/cache@v3
       with:
-        path:  mhlo-build
+        path: ${{ steps.setup_env.outputs.dependency_build_dir }}/mhlo-build
         key: ${{ runner.os }}-${{ runner.arch }}-mhlo-${{ needs.constants.outputs.mhlo_version }}-generic-build
         fail-on-cache-miss: True
 
     - name: Get Cached Enzyme Source
       id: cache-enzyme-source
       uses: actions/cache@v3
       with:
-        path:  mlir/Enzyme
+        path: ${{ steps.setup_env.outputs.dependency_build_dir }}/mlir/Enzyme
         key: Linux-enzyme-${{ needs.constants.outputs.enzyme_version }}-generic-source
         fail-on-cache-miss: True
 
     - name: Get Cached Enzyme Build
       id: cache-enzyme-build
       uses: actions/cache@v3
       with:
-        path:  enzyme-build
+        path: ${{ steps.setup_env.outputs.dependency_build_dir }}/enzyme-build
         key: ${{ runner.os }}-${{ runner.arch }}-enzyme-${{ needs.constants.outputs.llvm_version }}-${{ needs.constants.outputs.enzyme_version }}-generic-build
         fail-on-cache-miss: True
 
-    - name: Install OpenMP
-      run: |
-        brew install libomp
-
```

Comment:
Thanks! I've actually had issues building with libomp on my own m2 device. Since it appears you got it to work, what version of clang is available on the runner, and where was it installed from (brew or apple)?

Reply:
Here's the info on clang on the runner:
```
MBA-M7L99QWW0L:~ runner$ clang -v
Apple clang version 15.0.0 (clang-1500.1.0.2.5)
Target: arm64-apple-darwin23.1.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
MBA-M7L99QWW0L:~ runner$
MBA-M7L99QWW0L:~ runner$
```
So not brew installed.

That being said, libomp was indeed brew installed on the runner.

Reply:
Huh good to know, thanks!